src/main/groovy/org/labkey/gradle/plugin/Gwt.groovy

```diff
@@ -138,11 +139,11 @@ class Gwt implements Plugin<Project>
 
                         java.args =
                                 [
-                                        '-war', outputDir,
+                                        '-war', outputDirPath,
                                         '-style', project.gwt.style,
                                         '-logLevel', project.gwt.logLevel,
-                                        '-extra', extrasDir,
-                                        '-deploy', extrasDir,
+                                        '-extra', extrasDirPath,
+                                        '-deploy', extrasDirPath,
```

Comment:
These don't seem to be defined anywhere. Should they be `outputDir.path` and `extrasDir.path`?

Reply:
These are defined on lines 108 and 109.